devel/libjob_queue/src/job_queue_manager.c

```diff
@@ -78,10 +78,15 @@ int job_queue_manager_get_num_running( const job_queue_manager_type * manager) {
 }
 
 
-int job_queue_manager_get_num_complete( const job_queue_manager_type * manager) {
+int job_queue_manager_get_num_success( const job_queue_manager_type * manager) {
   return job_queue_get_num_complete( manager->job_queue );
```

Comment:
Shouldn't the method job_queue_get_num_complete in job_queue also be renamed to num_success? To support the clarifying "theme" of this PR regarding going from "Complete" to "Success/Failed"?